docs/sequence/error_scenarios/PISPLinkingGrantConsentPhase1.puml

```diff
@@ -0,0 +1,204 @@
+DFSP_TP_IN -> DFSP_LM: model.validateAuthToken()
+deactivate DFSP_TP_IN
+activate DFSP_LM
+
+DFSP_LM -> DFSP_LM: DFSPBackendRequests.validateAuthToken()
+rnote right of DFSP_LM #LightBlue
+Do we need two backend endpoints for validating
+web authTokens and OTP authTokens? Or is a DFSP expected to
+validate both cases with one endpoint?
+
+POST /validateAuthToken
+{
+  consentRequestId: '6789'
+  authChannel: model.data.authChannel,
+  authToken: '124356'
+}
+end note
+rnote right of DFSP_LM: state: authTokenValidated
+
+DFSP_LM -> DFSP_LM: const consentId = uuid()
+DFSP_LM -> DFSP_LM: model.grantConsent()
+DFSP_LM -> DFSP_LM: ThirdpartyRequests.postConsents()
+rnote right of DFSP_LM: state: consentGranted
+rnote right of DFSP_LM #LightBlue
+It's important to save the model with the consentId from this point onwards!
+Requests will not contain a consentRequestId in the upcoming requests.
+This may require changing `PersistentModel` to have a secondary key it can
+store the model with.
+end note
+DFSP_LM -> DFSP_LM: this.saveToKVS({key: '1a2b3c4d'})
+
+DFSP_LM -> Switch: POST /consents
+activate Switch
+
+rnote left of DFSP_LM #LightBlue
+POST /consents
+FSIOP-Source: dfspa
+FSIOP-Destination: pispa
+{
+  consentId: 1a2b3c4d
+  consentRequestId: 6789
+  scopes: model.data.scopes
+}
+end note
+Switch --> DFSP_LM: 202 ACCEPTED
+deactivate DFSP_LM
+Switch ->  PISP_TP_IN: POST /consents
+activate PISP_TP_IN
+PISP_TP_IN -> PISP_TP_IN: const model = await loadFromKVS({key: 6789})
+PISP_TP_IN -> PISP_LM: model.registerCredential()
+activate PISP_LM
+PISP_TP_IN --> Switch: 202 Accepted
+deactivate PISP_TP_IN
+deactivate Switch
+rnote right of PISP_LM: state: consentRecieved
+rnote right of PISP_LM #LightBlue
+It's important to save the model with the consentId from this point onwards!
+Requests will not contain a consentRequestId in the upcoming requests.
+This may require changing `PersistentModel` to have a secondary key it can
+store the model with.
+end note
+PISP_LM --> PISP_LM: const challenge = deriveChallenge(consentRequest)
+PISP_LM --> PISP_LM: this.saveToKVS({key: '1a2b3c4d'})
+
+PISP_LM -> PISP_LM: ThirdpartyRequests.putConsents()
+rnote right of PISP_LM: state: signedConsentSent
+PISP_LM -> Switch: PUT /consents/1a2b3c4d
+activate Switch
+
+rnote right of PISP_LM #LightBlue
+PUT /consents/1a2b3c4d
+FSIOP-Source: pispa
+FSIOP-Destination: dfspa
+{
+  scopes: [{
+    accountId: 'dfspa.username.1234',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+    accountId: 'dfspa.username.5678',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+  }],
+  credential: {
+    credentialType: "FIDO",
+    status: "PENDING",
+    payload: PublicKeyCredential
+  }
+}
+end note
+
+Switch --> PISP_LM: 202 Accepted
+deactivate PISP_LM
+Switch -> DFSP_TP_IN: PUT /consents/1a2b3c4d
+activate DFSP_TP_IN
+
+DFSP_TP_IN -> DFSP_TP_IN: const model = await loadFromKVS({key: 1a2b3c4d})
+rnote right of DFSP_TP_IN: state: signedConsentRecieved
```

Comment:
nit:  typo - should be `signedConsentReceived`

Reply:
thank you.